### PR TITLE
add use case for manually adding of data #FiBo-47 work 3h

### PR DESCRIPTION
**Pull request by XSnelliusX**

### CODE REVIEW

File: docs/SRS.md [hunk 1]
```diff
@@ -43,15 +43,23 @@ The project is going to be realized as an Android App.
 Planned Subsystems are:
 
 * Dashboard:  
-The dashboard is the main view of our user interface. It visualizes all the data that the user enters and offers the possibility to customize the view according to the user's wishes while also alow to categorize and filter the displayed data.
+  The dashboard is the main view of our user interface. It visualizes all the data that the user
+  enters and offers the possibility to customize the view according to the user's wishes while also
+  alow to categorize and filter the displayed data.
```
Comment (jens1o) [inline docs/SRS.md:48]:
```suggestion
  allow to categorize and filter the displayed data.
```

File: docs/SRS.md [hunk 2]
```diff
@@ -111,81 +132,134 @@ Deployment:
...
-This section will explain the different use cases, you could see in the Use Case Diagram, and their functionality.  
-Until December, we plan to implement:
+This section will explain the different use cases, you could see in the Use Case Diagram, and their
+functionality.  
+Until December we plan to implement:
+
+This section will explain the different use cases, you could see in the Use Case Diagram, and their
+functionality.  
+Until December we plan to implement:
+
```
Comment (jens1o) [inline docs/SRS.md:150]:
```suggestion
```

duplicate content

File: docs/SRS.md [hunk 3]
```diff
@@ -111,81 +132,134 @@ Deployment:
...
+In case you share your phone, have multiple accounts or just want to be cautious about your privacy
+you should be able to manually log out
+
+#### 3.1.4 Manual adding of data
 
-> TBD
+This is one of the essentials of our project. The user gets the possibility to manually add his
+expenses. Therefore the User has tho enter the store of the purchase, and the items with the
+respective deposited category and the price
```
Comment (jens1o) [inline docs/SRS.md:178]: As a follow-up ticket, we should create Use-Case Markdowns, where we can also embed the use-case diagram :)

File: docs/SRS.md [hunk 4]
```diff
@@ -111,81 +132,134 @@ Deployment:
...
+Until December we plan to implement:
+
+This section will explain the different use cases, you could see in the Use Case Diagram, and their
+functionality.  
+Until December we plan to implement:
+
+- 3.1.1 Create Account
+- 3.1.1 Logging in
+- 3.1.2 Logging out
+- 3.1.3 Manual adding of data
```
Comment (jens1o) [inline docs/SRS.md:154]: missing link to the markdown explaining that use-case :)

File: docs/use_cases/UC_4_manual_adding_of_data/UC_4_manual_adding_of_data.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+![Add receipt Button clicked](./add_button_clicked.png)
+
+
+### 2.1.3 Narrative
+
+```gherkin
+Feature: manual adding of data
+
+  As a signed in user
+  i want to add a receipt by manually entering the data
```
Comment (jens1o) [inline docs/use_cases/UC_4_manual_adding_of_data/UC_4_manual_adding_of_data.md:46]:
```suggestion
  I want to add a receipt by manually entering the data
```

File: docs/use_cases/UC_4_manual_adding_of_data/UC_4_manual_adding_of_data.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+  in order to track my spend money.
```
Comment (jens1o) [inline docs/use_cases/UC_4_manual_adding_of_data/UC_4_manual_adding_of_data.md:47]:
```suggestion
  in order to track my spent money.
```

**Review by jens1o — CHANGES_REQUESTED** [review]
see comments :) great work overall!

**Comment by XSnelliusX** [comment]
sollte alles gefixt sein

**Review by jens1o — APPROVED** [review]

**Pull request merged**